CHANGELOG.md

```diff
@@ -6,14 +6,18 @@ The change log describes what is "Added", "Removed", "Changed" or "Fixed" betwee
 
 ### Added
 
+- Service for`Http\Client\Common\PluginClientFactory`.
 - Any third party library using `Http\Client\Common\PluginClientFactory` to create `Http\Client\Common\PluginClient`
 instances now gets zero config profiling.
-- `Http\Client\Common\PluginClientFactory` factory service.
+- Add missing service reference for `cache_key_generator`
```

Comment:
i think this is a bugfix. its to make a configuration option that existed but was not working actually work as expected. ok if i move it and make it more explicit?

Reply:
oh, its already in the list of bugfixes for this version. i will remove this line.